src/libappimage/libappimage.cpp

```diff
@@ -28,27 +28,25 @@
 #endif
 
 using namespace appimage::core;
+using namespace appimage::utils;
 
 namespace bf = boost::filesystem;
 
-appimage::utils::Logger libappimageLogger("libappimage", std::clog);
-
 extern "C" {
 
 
 /* Check if a file is an AppImage. Returns the image type if it is, or -1 if it isn't */
 int appimage_get_type(const char* path, bool verbose) {
     typedef std::underlying_type<AppImageFormat>::type utype;
-
     try {
         AppImage appImage(path);
         return static_cast<utype>(appImage.getFormat());
     } catch (const std::runtime_error& err) {
         if (verbose)
```

Comment:
`verbose` should be ignored.

Reply:
This could wait until we decide to write the `verbose` function aliases.

Reply:
It should be ignored since we have a real logging system.